src/lib/Publisher.cs

```diff
@@ -1,43 +1,62 @@
 using System.Text.Json;
+using System.Text.RegularExpressions;
 
 namespace Microsoft.OpenApi.ApiManifest;
 
 public class Publisher
 {
     public string? Name { get; set; }
     public string? ContactEmail { get; set; }
+
     private const string NameProperty = "name";
     private const string ContactEmailProperty = "contactEmail";
 
-    public Publisher(string contactEmail)
+    private static readonly Regex s_emailRegex = new(@"^[^@\s]+@[^@\s]+\.[^@\s]+$", RegexOptions.Compiled, Constants.DefaultRegexTimeout);
```

Comment:
according to [the rfc](https://www.rfc-editor.org/rfc/rfc5322#section-3.4.1) .something is optional (i.e. me@local or bigcheese@com are valid email addresses)